### DISCUSSION TITLE
Do we request adding new boards as Issues/feature request?  Arducam Pico4ML would be a nice addition

### DISCUSSION THREAD

**Discussion by brianlmerritt** [earlephilhower/arduino-pico#1832] [Ideas]
It works using generic, but has a number of sensors such as camera, imu, microphone, plus a display on the back, and would make a nice addition to supported boards.

**Comment by earlephilhower** [comment]
Most times it's a user of the board itself, or the manufacturer, adding a board through a PR.  Especially when there are a ton of onboard peripherals, since you'll probably want to define special names for pins (i.e. `PIN_LCD_SPI` as opposed to just `D4`).  I myself, never having touched one, probably would not be the best person to do that. :)

If you'd like to take a stab at it, look at the closed PRs here.  There's lots of examples and [documentation](https://github.com/earlephilhower/arduino-pico/blob/master/docs/contrib.rst).

**Reply by brianlmerritt** [reply]
Thanks - I will check with Arducam and see if they have it in the works :D

**Comment by brianlmerritt** [comment]
I have raised the question here https://forum.arducam.com/t/pico4ml-support-in-arduino-ide-some-thoughts/6020 , will see if it progresses

**Comment by brianlmerritt** [comment]
Closing discussion
